### PR TITLE
Vital updates documentation

### PR DESCRIPTION
**Pull request by maruthiallam**
Description
----

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
Screen Shot 2021-06-17 at 4 11 02 PM
Screen Shot 2021-06-17 at 4 11 11 PM
Screen Shot 2021-06-17 at 4 11 40 PM
Screen Shot 2021-06-17 at 4 11 48 PM

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/diagnostic/observation.md [hunk 1]
```diff
@@ -246,9 +246,9 @@ Update an existing Observation.
 
 _Implementation Notes_
 
-* Currently only `laboratory` is supported.
+* Currently only `laboratory` and `vital-signs` is supported.
 * Both read and write scopes are required.
-* Updates on patient and category are not supported. Updating a value to a different value type is not supported.
+* Updates on patient and category are not supported.
 * See [FHIR<sup>®</sup> Update] for additional details about update operations.
```
Comment (joeratt) [inline content/millennium/r4/diagnostic/observation.md:252]: Can you include the note we included in creates about active relationship here too, please?

File: content/millennium/r4/diagnostic/observation.md [hunk 2]
```diff
@@ -246,9 +246,9 @@ Update an existing Observation.
 
 _Implementation Notes_
 
-* Currently only `laboratory` is supported.
+* Currently only `laboratory` and `vital-signs` is supported.
 * Both read and write scopes are required.
-* Updates on patient and category are not supported. Updating a value to a different value type is not supported.
+* Updates on patient and category are not supported.
```
Comment (joeratt) [inline content/millennium/r4/diagnostic/observation.md:251]: Can you clarify why this changed? I'm pretty sure this is still accurate.

Comment (fpivonka) [reply content/millennium/r4/diagnostic/observation.md:251]: @joeratt we fixed that for labs and support updates to change to different value types

Comment (joeratt) [reply content/millennium/r4/diagnostic/observation.md:251]: Do vitals updates support changing type, though?

Comment (fpivonka) [reply content/millennium/r4/diagnostic/observation.md:251]: @joeratt no they don’t. That’s what I get for looking at this on my phone 😂

File: lib/resources/r4/observation.yaml [hunk 3]
```diff
@@ -358,6 +387,7 @@ fields:
   note: |
     <ul>
       <li> Must be a reference to Practitioner. </li>
+      <li> Vital Signs require performer. </li>
```
Comment (joeratt) [inline lib/resources/r4/observation.yaml:390]: We might be able to leave this out. There's a change coming to remove this restriction

File: content/millennium/r4/diagnostic/observation.md (hunk 2 above)
Comment (joeratt) [reply content/millennium/r4/diagnostic/observation.md:251]: I see the "labs only" limitation called out here: https://github.com/cerner/fhir.cerner.com/pull/675/files#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR640

I'm not sure why we have 2 different sections communicating the same info. @maruthiallam  - can you help make sure they're in-sync, please?

Comment (maruthiallam) [reply content/millennium/r4/diagnostic/observation.md:251]: The idea is updating value type to the same value type vs updating to other value type.  I have updated it to look like the  below screenshot. Let me know if it's still confusing.
Screen Shot 2021-06-18 at 9 24 55 AM

File: content/millennium/r4/diagnostic/observation.md (hunk 1 above)
Comment (maruthiallam) [reply content/millennium/r4/diagnostic/observation.md:252]: Screen Shot 2021-06-18 at 9 28 26 AM

File: lib/resources/r4/observation.yaml (hunk 3 above)
Comment (maruthiallam) [reply lib/resources/r4/observation.yaml]: https://github.com/cerner/fhir.cerner.com/pull/675/commits/52e09da54bd018cba858d003d63e103e1bf4a2a9#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcL390

**Review by joeratt — APPROVED** [review]

File: content/millennium/r4/diagnostic/observation.md (hunk 2 above)
Comment (lalitha94) [reply content/millennium/r4/diagnostic/observation.md:251]: nit: Can we put the labs one together and have something like
Laboratory supports `valueQuantity`, `valueCodeableConcept`, `valueString`.
Laboratory also, supports updating observations with 'valueDateTime' to 'valueDateTime' type.
Vital Signs support only updating observations with `valueQuantity` to `valueQuantity` type

Any thoughts?

**Review by lalitha94 — APPROVED** [review]

File: content/millennium/r4/diagnostic/observation.md (hunk 2 above)
Comment (lalitha94) [reply content/millennium/r4/diagnostic/observation.md:251]: Discussed offline and decided that it better to have it based on type.

File: content/millennium/r4/diagnostic/observation.md [hunk 4]
```diff
@@ -246,15 +246,19 @@ Update an existing Observation.
 
 _Implementation Notes_
 
-* Currently only `laboratory` is supported.
+* Currently only `laboratory` and `vital-signs` is supported.
```
Comment (kathypickering) [inline content/millennium/r4/diagnostic/observation.md:249]: change 'is' to 'are'

File: content/millennium/r4/diagnostic/observation.md (hunk 2 above)
Comment (kathypickering) [reply content/millennium/r4/diagnostic/observation.md:251]: The exception is valueDateTime cannot be updated to **another** value type...

Comment (maruthiallam) [reply content/millennium/r4/diagnostic/observation.md:251]: Screen Shot 2021-06-18 at 10 55 02 AM

File: content/millennium/r4/diagnostic/observation.md (hunk 4 above)
Comment (maruthiallam) [reply content/millennium/r4/diagnostic/observation.md]: [commit](https://github.com/cerner/fhir.cerner.com/pull/675/commits/79d5eab8c4b005269538ab07cfc833058b9698f5#diff-fbd7021b360068e6d7f7f8ef265148e4572c35c1766f37c4280d8833b3c24749R249)

**Review by kathypickering — APPROVED** [review]

**Comment by ebbp123** [comment]
Verified live:
image (5)

**Pull request merged**